### PR TITLE
update code

### PR DESCRIPTION
**Pull request by tier5aces5**

### CODE REVIEW

File: ujjal.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # Test file to check PR
```
Comment (DSubhadipSahoo) [inline ujjal.md]: Change this title to gitpoc

File: ujjal.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+- again changed
```
Comment (DSubhadipSahoo) [inline ujjal.md]: Make a in caps

**Review by DSubhadipSahoo — CHANGES_REQUESTED** [review]
Please make the changes suggested

File: ujjal.md (hunk 1 above)
Comment (tier5aces5) [reply ujjal.md]: changed

File: ujjal.md (hunk 2 above)
Comment (tier5aces5) [reply ujjal.md]: gitpoc

**Review by DSubhadipSahoo — APPROVED** [review]

**Pull request merged**